docs/install.md

```diff
@@ -23,9 +23,12 @@ Unpack the Draft binary and add it to your PATH.
 To install the server-side of Draft, use `draft init` with your ingress' `basedomain` and credentials to let Draft communicate with a Docker registry by using the following command:
 
 ```
-$ draft init --set registry.url=docker.io,registry.org=changeme,registry.authtoken=changeme,basedomain=changeme
+$ draft init --set registry.url=changeme,registry.org=changeme,registry.authtoken=changeme,basedomain=changeme
 ```
 
+* registry.url: Docker Registry Server URL. e.g. Azure Container Registry -> xxxx.azurecr.io
```

Comment:
can you also add this to the end of this sentence?

> e.g. Azure Container Registry -> xxxx.azurecr.io, DockerHub -> docker.io

I feel like lots of people using DockerHub will supply an empty field, which will trip up draft. Just making it explicit here will help. :)